scripts/add_changelog

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+import textwrap
+from typing import List, Optional, Tuple
+
+def popen_full(cmd: List[str]) -> Tuple[str, str, int]:
+  "Runs `cmd` and returns (stdout, stderr, exit_code)"
+  process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+  stdout, stderr = process.communicate()
+  exit_code = process.wait()
+  return stdout.decode(), stderr.decode(), exit_code
+
+def run_or_die(cmd: str) -> None:
+  "Runs `cmd` and dies if it fails."
+  exit_code = os.system(cmd)
+  if exit_code != 0:
+    sys.exit(exit_code)
+
+def find_pr(branch: str) -> Optional[str]:
+  "Returns the PR number for the given branch, or None if no PR exists."
+  pr, error, exit_code = popen_full(['gh', 'pr', 'view', '--json', 'number', '--jq', '.number', branch])
```

Comment:
Maybe we could add something in the docstring about `gh` being a prerequisite? I assume it's just `brew install gh`?

Reply:
I added a more informative error message (a few lines down) and I added a whole big new commit that refreshes top-level README.md. We had a section on development tools/dependencies there, and I expanded it now. Not just with `gh`; there are several other tools we're using too.

Reply:
thanks!